clang/tools/c-index-test/JSONAggregation.cpp

```diff
@@ -270,12 +298,16 @@ void Aggregator::dumpJSON(raw_ostream &OS) {
   for (unsigned i = 0, e = Symbols.size(); i != e; ++i) {
     OS.indent(4) << "{\n";
     SymbolInfo &symInfo = Symbols[i];
-    OS.indent(6) << "\"kind\": \"" << getSymbolKindString(symInfo.Kind) << "\",\n";
-    OS.indent(6) << "\"lang\": \"" << getSymbolLanguageString(symInfo.Lang) << "\",\n";
-    OS.indent(6) << "\"usr\": \"" << symInfo.USR << "\",\n";
-    OS.indent(6) << "\"name\": \"" << symInfo.Name << "\",\n";
+    OS.indent(6) << "\"kind\": \""
+                 << escapeForJson(getSymbolKindString(symInfo.Kind)) << "\",\n";
```

Comment:
I think `dumpJSON()` should be using `llvm::json::OStream` instead of emitting this by hand.

https://llvm.org/doxygen/classllvm_1_1json_1_1OStream.html

Reply:
That's probably a better approach for this file overall, but I tried to keep this PR as small as possible. I've switched to `llvm::json::Value` as @drodriguez suggested.